src/app/core/utils.py

```diff
@@ -0,0 +1,2 @@
+def create_examples(data: dict) -> dict:
```

Comment:
as mentioned elsewhere, i would just remove the "value" layer and get rid of this file entirely

Reply:
The value layer has its use due to FastAPI's way of creating the OpenAPI specs. I removed this function however and inlined it.